src/components/input/input.js

```diff
@@ -634,11 +634,25 @@ function mdMaxlengthDirective($animate, $mdUtil) {
   };
 
   function postLink(scope, element, attr, ctrls) {
-    var maxlength;
+    var maxlength = Number.parseInt(attr.mdMaxlength);
```

Comment:
As pointed out by Jeremy, `Number.parseInt` is not supported in IE11. So we'll need to use something different here.

It looks like the global [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) should work?
MDN also points out that [Number.parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt) has no IE support.

Reply:
I've used the global parseInt instead